openmp/runtime/src/ompt-general.cpp

```diff
@@ -23,6 +23,7 @@
 #if KMP_OS_UNIX
 #include <dlfcn.h>
 #endif
+#include "kmp_utils.h"
```

Comment:
put it in the right order?

Reply:
> put it in the right order?

Could you please be more specific? @shiltian

Reply:
@DanielCChen, I think https://llvm.org/docs/CodingStandards.html#include-style says that this header should be moved up. Indeed, it wouldn't fit under the "system include files" comment.

I don't know what to say about the include of `ompt-specific.cpp` (yes, `.cpp`!) below.

Reply:
Yes. That is what I meant.

> I don't know what to say about the include of ompt-specific.cpp (yes, .cpp!) below.

Me neither. ;-) That is before `libomp` was contributed to LLVM.

Reply:
Thanks. I will move it before the system #include then.